ajax/libs/Leaflet.EasyButton/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "leaflet-easybutton",
+  "description": "easily add control buttons to your leaflet maps with icon support",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CliffCloud/Leaflet.EasyButton.git"
+  },
+  "keywords": [
+    "leaflet",
+    "font-awesome",
+    "glyphicon",
+    "button",
+    "control",
+    "map"
+  ],
+  "author": "rocks.in.the.cloud+easybutton@gmail.com",
+  "license": "MIT",
+  "homepage": "https://github.com/CliffCloud/Leaflet.EasyButton",
```

Comment:
Please don't forget to remove the `,` from the last field for valid json format, thanks.